pkg/api/types.go

```diff
@@ -1802,7 +1826,25 @@ const MaxSecretSize = 1 * 1024 * 1024
 type SecretType string
 
 const (
-	SecretTypeOpaque SecretType = "Opaque" // Default; arbitrary user-defined data
+	// SecretTypeOpaque is the default; arbitrary user-defined data
+	SecretTypeOpaque SecretType = "Opaque"
+
+	// SecretTypeServiceAccountToken contains a token that identifies a service account to the API
```

Comment:
Secrets with this type are automatically managed/gc'ed by the ServiceAccountController/TokenController.  Opaque type tokens are not automatically managed by any controller.